Versionen 1 und 2/HUSST_Ergebnisdaten_2_21.xsd

```diff
@@ -0,0 +1,2855 @@
+						verwendet werden: eindeutige Kennung der Netzversion
+						(Basisversion)
+					</documentation>
+				</annotation>
+			</element>
+			<element name="Aktion" type="kts:ktFahrtVerlaufAktion"
+				minOccurs="1" maxOccurs="1">
+			</element>
+			<element name="Linie" type="int" maxOccurs="1" minOccurs="0">
+			</element>
+			<element name="FahrtNr" type="int" minOccurs="0"
+				maxOccurs="1">
+			</element>
+			<element name="Messfahrt" type="boolean" minOccurs="0"
+				maxOccurs="1">
+			</element>
+			<element name="OpIdx" type="int" maxOccurs="1" minOccurs="0">
+				<annotation>
+					<documentation>
+						Index der aktuellen bzw. gerade verlassenen
+						Haltestelle (beginnend ab 1 für die erste
+						Haltestelle der Fahrt)
+					</documentation>
+				</annotation>
+			</element>
+			<element name="OpTyp" type="int" maxOccurs="1" minOccurs="0">
+				<annotation>
+					<documentation>
+						Ortspunkttyp (z.B. 1=Hst, 2=Betriebshof)
+					</documentation>
+				</annotation>
+			</element>
+			<element name="OpNr" type="int" maxOccurs="1" minOccurs="0">
+				<annotation>
+					<documentation>Orstpunktnummer</documentation>
+				</annotation>
+			</element>
+			<element name="Tacho" type="int" maxOccurs="1" minOccurs="0">
+				<annotation>
+					<documentation>Wegzählerstand</documentation>
+				</annotation>
+			</element>
+			<element name="Distanz" type="int" maxOccurs="1"
+				minOccurs="0">
+				<annotation>
+					<documentation>
+						Entfernung ab der Haltestelle in Metern
+					</documentation>
+				</annotation>
+			</element>
+			<element name="Geo" type="kts:ktGeo" maxOccurs="1"
+				minOccurs="0">
+			</element>
+			<element name="TSoll" type="dateTime" maxOccurs="1"
+				minOccurs="0">
+				<annotation>
+					<documentation>Sollzeit</documentation>
+				</annotation>
+			</element>
+			<element name="Fahrplanlage" type="int" maxOccurs="1"
+				minOccurs="0">
+				<annotation>
+					<documentation>
+						Fahrplanlage als vorzeichenbehaftete Abweichung
+						in
+						Sekunden
+					</documentation>
+				</annotation>
+			</element>
+			<element name="LeistungArt" type="int" maxOccurs="1"
+				minOccurs="0">
+			</element>
+			<element name="Umlauf" type="int" maxOccurs="1" minOccurs="0">
+				<annotation>
+					<documentation>Wagenumlauf</documentation>
+				</annotation>
+			</element>
+			<element name="Route" type="int" maxOccurs="1" minOccurs="0">
+				<annotation>
+					<documentation>Linienvariante</documentation>
+				</annotation>
+			</element>
+			<element name="ZaehldatenListe" type="kts:ktZaehldatenListe"
+				maxOccurs="1" minOccurs="0">
+			</element>
+		</sequence>
+		<attribute name="DatenVersion" type="int" use="optional">
+			<annotation>
+				<documentation>
+					Die Datenversion setzt sich aus einem zweigeteilten
+					Inhalt zusammen:
+					(ID_Inhaltsversion &lt;&lt; 8) | ID_Zeitraum wobei
+					0 &lt;
+					ID_Zeitraum &lt; 256
+					gilt. Die bisher genutzte Netzversion
+					wird damit obsolet.
```

Comment:
```suggestion
					gilt.
```
Ich glaube den Hinweis auf die Netzversion brauchen wir hier auch nicht mehr, in den anderen Kommentaren zur Datenversion ist dieser Hinweis auch schon nicht mehr enthalten und die Netzversion selber verweist ja schon auf die Datenversion als neues Attribut.